features/steps/test_starter/{{ cookiecutter.repo_name }}/src/requirements.txt

```diff
@@ -7,7 +7,6 @@ jupyter~=1.0
 jupyterlab_server>=2.11.1, <2.16.0
 jupyterlab~=3.0, <3.6.0
 kedro[pandas.CSVDataSet]=={{ cookiecutter.kedro_version }}
-kedro-telemetry~=0.2.0
```

Comment:
Maybe worth restoring this to see if the tests pass with the conditional pinning

Reply:
They don't, I tried it yesterday on this PR https://github.com/kedro-org/kedro/pull/2993/files

Reply:
Thanks, hadn't seen it 👍